backend/tracim_backend/views/collaborative_document_edition_api/collaborative_document_edition_controller.py

```diff
@@ -0,0 +1,154 @@
+from http import HTTPStatus
```

Comment:
this file should be `controller.py` only

Reply:
done